src/routes/(split)/components/page-header/+page.md

```diff
@@ -0,0 +1,137 @@
+<script lang="ts">
+    import PageHeader from "$lib/components/PageHeader.svelte";
+    import Card from "$lib/components/Card.svelte";
+    import icpLogo from "$lib/assets/icp-rounded.svg";
+
+</script>
+
+# Page Header
+
+Used for settings and detail pages to have a first overview of the entity.
+
+```html
+<PageHeader>
+  <span slot="topLeft">Top left</span>
+  <span slot="topRight">Top right</span>
+  <h1 slot="title">Title</h1>
+  <h3 slot="subTitle">This page header is inside a card</h3>
+</PageHeader>
+```
+
+## Properties
+
+| Property | Description                                                     | Type                    | Default     |
+| -------- | --------------------------------------------------------------- | ----------------------- | ----------- |
+| `testId` | Add a `data-tid` attribute to the DOM, useful for test purpose. | `string` or `undefined` | `undefined` |
+
+## Slots
+
+| Slot name  | Description                                        |
+| ---------- | -------------------------------------------------- |
+| `title`    | The main information to render.                    |
+| `subTitle` | Secondary information to render.                   |
+| `topLeft`  | Additional information to render at the top left.  |
+| `topRight` | Additional information to render at the top right. |
+
+## Showcase
+
+The component is within a `div` with the background to highlight the component exactly as it is.
+
+**IMPORTANT: The component has no internal padding. The spacing with the other components is responsibility of the parent using this component.**
+
+### Simple example:
+
+<div class="wrapper">
+  <PageHeader>
+    <span slot="topLeft">Top left</span>
+    <span slot="topRight">Top right</span>
+    <h1 slot="title">Title</h1>
+    <h3 slot="subTitle">This page header is inside a card</h3>
+  </PageHeader>
+</div>
+
+### Real production example
+
+<div class="wrapper">
+  <PageHeader>
+    <span slot="topLeft">
```

Comment:
should it rather be a `svelte:fragment`?

Reply:
This is an example, I used a `span`. It depends on who uses it.

Reply:
Kind of feel it would be cleaner to provide example with fragments.

The component has a structure and is opiniated, so the default should be to use it as it without using sub element when not needed.

Not a strong opinion though. Thanks for the answer.